web/src/js/index.js

```diff
@@ -1,473 +1,479 @@
+          });
+        }
+      }
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+        titleColor: "white",
+      });
+    });
+}
+
+// Get allowlist
+function getAllowlist() {
+  app.allowlist = [];
+  app.showAllowlistLoadingGif = true;
+  fetch(API_GET_ALLOWLIST, {
+    headers: {
+      "x-api-key": API_KEY,
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      let i = 1;
+      let allowlistRaw = data["response"]["allowlist"];
+
+      dayjs.extend(dayjs_plugin_utc);
+      dayjs.extend(dayjs_plugin_timezone);
+
+      app.allowlist = allowlistRaw.map((item) => {
+        // Parse Resource ID, i.e. split on ":"
+        let parsedResourceId = item["resource_id"].split(":", 2);
+        parsedResourceId.push(
+          item["resource_id"].slice(parsedResourceId.join("").length + 2)
+        );
+
+        let readableDate = dayjs.unix(item["expiration"]).tz(dayjs.tz.guess());
+
+        item["row_id"] = i++;
+        item["service"] = parsedResourceId[0];
+        item["resource"] = parsedResourceId[1];
+        item["id"] = parsedResourceId[2];
+
+        item["expiration_readable"] = readableDate.format(
+          "ddd MMM DD HH:mm:ss YYYY"
+        );
+
+        return item;
+      });
+
+      setTimeout(function () {
+        app.allowlistDataTables = $("#allowlist").DataTable({
+          columnDefs: [
+            { className: "dt-center", targets: [5] },
+            { orderable: false, targets: [0, 1, 2, 3, 4, 5, 6, 7] },
+            {
+              targets: [6],
+              visible: false,
+              searchable: false,
+            },
+          ],
+          dom: "rtp",
+          order: [[6, "desc"]],
+          pageLength: 20,
+          rowGroup: {
+            dataSrc: 6,
+          },
+        });
+      }, 10);
+
+      app.showAllowlistLoadingGif = false;
+    })
+    .catch((error) => {
+      iziToast.error({
+        color: "#EC2B55",
+        message: error,
+        messageColor: "white",
+        title: "Something went wrong",
+      });
+    });
+}
+
+function refreshAllowlist() {
+  app.allowlistDataTables.destroy();
+  getAllowlist();
+}
+
+function openTab(evt, tabName) {
+  var i, x, tablinks;
+
+  x = document.getElementsByClassName("content-tab");
+  for (i = 0; i < x.length; i++) {
+    x[i].style.display = "none";
+  }
+
+  tablinks = document.getElementsByClassName("tab");
+  for (i = 0; i < x.length; i++) {
+    tablinks[i].className = tablinks[i].className.replace(" is-active", "");
+  }
+
+  document.getElementById(tabName).style.display = "block";
+  evt.currentTarget.className += " is-active";
+}
+
+function init() {
+  getAllowlist();
+  getExecutionLogList();
+  getServices();
+}
+
+// Get the API Gateway Base URL from manifest file
+fetch("serverless.manifest.json").then(function (response) {
```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/servian/aws-auto-cleanup/pullRequest?prid=9792602)
```suggestion
fetch("serverless.manifest.json").then(function (response) {
```